web/src/routes/execution.tsx

```diff
@@ -7,28 +15,39 @@ import {
   useParams,
 } from "react-router"
 import { ReactElement, useEffect, useState } from "react"
-import {
-  getExecutionStatus,
-  changeExecutionStatus,
-  togglePlayerStatus,
-} from "../api"
 import _ from "lodash"
 import { config } from "../config"
 import {
   Player,
   ExecutionData,
   isExecutionData,
-  ExecutionStatusEnum,
+  Role,
+  Location,
+  ExecutionStatus,
 } from "../types"
 import CsrfForm from "../components/CsrfForm"
 import { useSubmit } from "react-router-dom"
+import { changeExecutionStatus, getExecution, togglePlayerStatus, createNewPlayer, } from "../api"
 
-function TanCard({ tan }: { tan: string }): ReactElement {
+function TanCard({ player }: { player: Player }): ReactElement {
```

Comment:
For this component, we could utilize `<details>` & `<summary>`. 
I will probably add this in a future PR.